carmin.yaml

```diff
@@ -371,14 +392,14 @@ definitions:
   Execution:
     type: object
     required:
-      - identifier
```

Comment:
```
/executions/create-and-start
```
This method needs to return execution identifier otherwise we cannot get the status of Execution.